README.md

```diff
@@ -53,8 +50,7 @@ To contribute a patch, follow these instructions (note that installing
 ```
 
 
-License (see also LICENSE.txt)
-==============================
+## License (see also LICENSE.txt)
```

Comment:
Hi @chrismattmann What I had done on README.md was applying **# Heading 1** (markdown heading styles) for all the headings replacing 

=============================================================
 Welcome to Apache OODT  <http://oodt.apache.org/>
=============================================================
type headings. As I saw, you had fixed the main heading (Apache OODT heading)'s formatting with **# Heading**  style. Now the formatting of README is ok and my previous suggestion is not needed.